Work/work.py

```diff
@@ -0,0 +1,85 @@
+print( "WELCOME TO THE PHONEBOOK DIRECTORY") 
+ 
+
+filename = "myphonebook.txt" 
+myfile = open(filename, "a+") 
+myfile.close 
+ 
+ 
+def main_menu(): 
+    print( "\nMAIN MENU\n") 
+    print( "1. Show all existing Contacts") 
+    print( "2. Add a new Contact") 
+    print( "3. Search the existing Contact") 
+    print( "4. Exit") 
```

Comment:
Можно обойтись одним вызовом функции print
```suggestion
    print("\nMAIN MENU\n\n"
              "1. Show all existing Contacts\n"
              "2. Add a new Contact\n"
              "3. Search the existing Contact\n"
              "4. Exit")
```